refs/schemas/priv/expected-behavior.md

```diff
@@ -0,0 +1,655 @@
+   > All such Data Captures objects are of interest to the Privacy Compiler.
+
+   - *All Requests*: a list of all the [Privacy Request](./RFC-PRIV.md#privacy-request) objects that the Privacy Compiler is aware of
+   - *All Responses*: a list of all the [Privacy Request Response](./RFC-PRIV.md#privacy-request-response) objects that the Privacy Compiler is aware of
+   - Legal Bases:
+       - *All Consents*: a list of all the [Consent](./RFC-PRIV.md#consent) objects that the Privacy Compiler is aware of
+       - *All Contracts*: establishing a correspondence between a Data Subject and a `contract-id` being an identifier of a particular relationship that the Data Subject has with the system (e.g. a user account open with the system, employment contract, a court case treated by a lawfirm, etc.) with a `start` date and (optionally, if ended) a `end` date.
+       - *All Legitimate Interests*: establishing a correspondence between a Data Subject and each [Privacy Scope Triples](#privacy-scope-triples) included in the **Intended Privacy Scope** under `LEGITIMATE-INTEREST`
+       - *Necessary*: establishing a correspondence between a Data Subject and each [Privacy Scope Triple](#privacy-scope-triples) included in the **Intended Privacy Scope** under `NECESSARY` Legal Base
+
+- **Runtime Maps**, updated at runtime:
+    - Active Legal Bases:
+        - *Active Consents*: a list of `consent-id`s that is modified when Consent is collected and within [Operations over consents](#operations-over-consents)
+        - *Active Contracts*: a list of `contract-id`s of all active contracts that have not been subject to a `RELATIONSHIP-END` event
+        - *Active Legitimate Interests*: a subset of *All Legitimate Interests*
+    - Events, for [Resolving Retention Policies](#resolving-retention-policies):
+        - `SERVICE-END` events for contracts that end (e.g. user closes the account, or stops a service agreement), that directly impact *Active Contracts* the [Privacy Scope Triples](#privacy-scope-triples) of which have not been in the scope of any `DELETE`, `OBJECT`, or `RESTRICT` Privacy Request by the data Subject concerned
+        - `RELATIONSHIP-END` events the Data Subject ended all contracts and ended the relationship with the System/Organization
+        - `CAPTURE-DATE` when a Data Capture is made or updated
+    - Transfers:
+        - All the data exchanges with other Systems, as described in [Remembering Transfers](./RFC-PRIV.md#remembering-transfers).
+    - Privacy Scope:
+        - For each Data Subject, an **Eligible Privacy Scope**, according to [Privacy Algebra](#privacy-algebra).
+        This **Eligible Privacy Scope** is easily resolved at the level of each [Data Capture Fragment](./RFC-PRIV.md#data-capture-fragments) (or its `selector`)
+
+## Privacy Scope Triples
+
+A **<a name="triple"></a>Privacy Scope Triple** is a unit of [Privacy Scope](RFC-PRIV.md#privacy-scope)
+and it consists of (in that order):
```

Comment:
From the [RFC-PRIV](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#privacy-scope) document, each element of a triple has cardinality `0-*`.
Here, it's `one`.

Which is true?

Reply:
Both are true. 
Privacy Scope != Privacy Scope Triple

Privacy Scope can have mutiple values per dimension. This translates to a multitude of Triples (each monovalent per dimension) covering all combinations from the Privacy Scope.